edx_lint/files/pylintrc

```diff
@@ -36,28 +249,93 @@ load-plugins=edx_lint.pylint,pylint_django,pylint_celery
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
 disable=
+    # These should be left to the discretion of the reviewer
```

Comment:
Do we have statistics on how often each of these checks is either violated or locally disabled?  And is there a significant increase in runtime from enabling them?